changelog.rst

```diff
@@ -14,6 +14,10 @@ Check https://github.com/conan-io/conan for issues and more details about develo
 1.6.0 ()
 ----------------------
 
+- Feature: Added new ``self.run(..., run_environment=True)`` argument, that applies automatically PATH, LD_LIBRARY_PATH and DYLD_LIBRARY_PATH
+  environment variable, from the dependencies, to the execution of the current command.
+- Feature: Added new ``self.run(..., ignore_errors=True)`` argument, that inhibits launching an Exception if the commands fails, user can
```

Comment:
- ``exception``
- What does "return code" mean? Would it be the exception object or just the error message?